src/components/FilterPanel/FilterPanelSearch/__tests__/FilterSearch.spec.js

```diff
@@ -84,16 +86,13 @@ describe('FilterPanelSearch', () => {
     expect(getByTestId('result-content')).toBeVisible();
   });
 
-  test('invokes onChange when the user types a value into the search', () => {
```

Comment:
Do you know why this test is getting removed?

Reply:
Great question and I'd love your thoughts on this - This is related to https://github.com/carbon-design-system/carbon/issues/7481 , in which the `FilterPanelSearch` unit test is failing because of a defect in the underlying `Search`.

I looked at the `FilterPanelSearch` source code and the `onChange` is spread via the `other` additional props - https://github.com/carbon-design-system/ibm-security/blob/dev/src/components/FilterPanel/FilterPanelSearch/FilterPanelSearch.js#L32

Here's what I'm wondering - should the `FilterPanelSearch` unit test only be concerned with successfully passing through additional props, rather than testing the `onChange` handler of the `Search` component? This feels like something the `Search` unit tests should be concerned with.